kolibri/core/errorreports/middleware.py

```diff
@@ -23,11 +48,19 @@ def __call__(self, request):
     def process_exception(self, request, exception):
         error_message = str(exception)
         traceback_info = traceback.format_exc()
+        context = {
+            "request_info": get_request_info(request),
+            "server": get_server_info(request),
+            "packages": get_packages(),
+            "python_version": get_python_version(),
```

Comment:
Can we also add the `installer` -- which gives an indication to which installation type was used;

https://github.com/learningequality/kolibri/blob/24186194c23f2f0dd126e72bdc79ed4b5ddd413f/kolibri/core/device/api.py#L201